CDCgov/prime-simplereport
# Dan/6291 Patient form unknown fields

# FRONTEND PULL REQUEST

## Related Issue

- resolves #6291 

## Changes Proposed

- Allow users to indicate that certain fields (address, phone number) are unknown, skipping validation for those fields on create/edit of a patient.

## Additional Information

- Change includes some minor backend changes as well

## Testing

- Select checkbox on a form where fields are populated - verify that data was appropriately cleared out.
- Deselect checkbox and verify validation triggers correctly.
Screens:
- New user creation form
- Edit existing user
- Patient self-registration

## Screenshots / Demos

- TODO
